docs/release-notes/platform.rst

```diff
@@ -41,3 +41,5 @@ Platform Notes
   * Slurm
   * Cray XE, XC, and XK
   * Oracle Grid Engine (OGE) 6.1, 6.2 and open source Grid Engine
+
+.. note:: There is a known issue with the HWLOC releases older than 2.8.0 and systems with Ponte Vecchio accelerators.  HWLOC 2.8.0 or newer is required on such systems.
```

Comment:
This doesn't feel like quite the right location to put this information.  How about adding either in the table at the top of `installing-open-mpi/required-support-libraries.rst`, or perhaps in a "Note" right below the table...?

```suggestion
.. note:: There is a known issue with the [Hwloc](https://www.open-mpi.org/projects/hwloc/) releases older than v2.8.0 and systems with Intel Ponte Vecchio accelerators.  Hwloc v2.8.0 or newer is required on such systems.
```

Reply:
okay that sounds good.  i'll put the blurb there.